src/assets/scripts/client/constants/globalConstants.js

```diff
@@ -31,7 +31,7 @@ export const REGEX = {
     ALTITUDE_RESTRICTION: /^A([0-9]{1,3})([+-]?)$/,
     COMPASS_DIRECTION: /^[NESW]/,
     DOUBLE_DOT: /\.\./g,
-    HOLD_DISTANCE: /^([1-9]|1[0-9])(nm|min)$/i,
+    HOLD_DISTANCE: /^(0(?=\.)|[1-9]|1[0-9])(((?<!0)\.0|\.[1-9]))?(nm|min)$/i,
```

Comment:
This is some pseudo-regex (?) for understanding what this is doing.

1. Match beginning of string
2. Match a 0 if it is followed by a dot, otherwise match 1 to 9 or 10 to 19
3. Match .0 if it is not preceded by a 0, otherwise match .1 to .9
4. Match nm or min
5. Match end of string
6. Option to match nm and min without case-sensitivity

Reply:
This accomplishes your goal of allowing one decimal place for lengths in `min`, but inadvertently allows the same for `nm` as well. Do we really want that? Are there use cases to support that?

Reply:
I know the regex is currently only used to test and not match, but for future-proofing suggest that it should have two capturing groups. The first should capture the entire numeric portion, and the second containing the units. Use non-capturing groups for everything else.

Reply:
I'll defer to @erikquinn on if we actually want that or not. He can say whether or not thats actually a thing.

I'm going to guess fractional `nm` probably isn't a thing and, thus, I love the idea of multiple capture groups.  I'm also a fan of make future us happy because we did things in a flexible way today 😉 

I like your plans here!